### PR TITLE
offline_sev_kbc: fix compatabiliy with sample kbs

### PR DESCRIPTION
**Pull request by fitzthum**
Annotation format got out of sync between `offline_sev_kbc` and `offline_fs_kbs`. We have previously discussed more effective code sharing between these two and maybe now is the time to implement. I am posting this simple fixup to document components needed for SEV demo.

### CODE REVIEW

**Review by Jakob-Naucke — APPROVED** [review]

**Comment by fitzthum** [comment]
I'll fix the tests shortly.

**Comment by fitzthum** [comment]
Looks like we have a bit of a CI issue.

File: src/kbc_modules/offline_sev_kbc/mod.rs [hunk 1]
```diff
@@ -26,9 +26,9 @@ pub struct AnnotationPacket {
     // Key ID to manage multiple keys
     pub kid: String,
     // Encrypted key to unwrap
-    pub wrapped_data: Vec<u8>,
+    pub wrapped_data: String,
     // Initialisation vector
-    pub iv: Vec<u8>,
+    pub iv: String,
```
Comment (dubek) [inline src/kbc_modules/offline_sev_kbc/mod.rs:31]: Consider adding `(base64-encoded)` to both comments above the fields (if I understand correctly, the `String` is for base64-string).

**Review by dubek — APPROVED** [review]
Looks good. 
Small comment below (can be modified later):

**Comment by dubek** [comment]
Now that #34 is merged, hopefully rebasing this on top of `main` will solve the two CI errors with rustfmt on nightly and beta.

@fitzthum when you're back — please rebase...

**Comment by sameo** [comment]
> Now that #34 is merged, hopefully rebasing this on top of `main` will solve the two CI errors with rustfmt on nightly and beta.
> 
> @fitzthum when you're back — please rebase...

I just rebased and pushed @fitzthum branch.

**Review by sameo — APPROVED** [review]

**Comment by dubek** [comment]
Thanks @sameo !

**Pull request merged**